exercicios/para-casa/entregas/priscila-ferreira-de-sousa/exercicioParaCasa.js

```diff
@@ -144,4 +37,20 @@ console.log(Banco.bancosCriados);
 console.log("\n\n Removendo um banco associado ao cliente1...");
 cliente1.removeBank(banco1);
 console.log(banco1);
-console.log(cliente1);
\ No newline at end of file
+console.log(cliente1);
+
+const contaJuliaSantander = new ContaBanco(cliente1, banco2, 1234, 072);
```

Comment:
aqui ele me retornou o seguinte erro no terminal: 
`SyntaxError: Octal literals are not allowed in strict mode.`
Como essas infos são referentes ao numero e agencia da conta, que são números 'fixos' que você não faz conta de matemática propriamente falando, a prof disse que seria melhor que colocasse em string. E de fato, quando mudei pra string o código não deu mais esse erro.